### PR TITLE
Adds recover phone blog.

### PR DESCRIPTION
**Pull request by amitg01**

### CODE REVIEW

**Comment by amitg01** [comment]
This branch needs to be rebased with master after PR #4 is merged.

**Comment by n1c01a5** [comment]
The font size and the spacing of the article is not good.
For example take the medium.com as example in the smartphone format https://medium.com/macoclock/the-future-of-the-mac-9eed4d188b5 for the alignment, the font size and the spacing.

**Comment by bhavesh92pandey** [comment]
Hi, I have made changes to Blog 1 and 2 for headings and subheadings

**Comment by amitg01** [comment]
@bhavesh92pandey I have made the changes. Please review.

File: pages/blog/how-can-i-recover-my-phone.js [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+export default function RecoverPhoneBlog() {
+  return (
+    <Layout>
+      <Head>
+        <title>Recover.ws - Loser Box to protect your item from loss</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <div className="recover_phone_container">
+        <section className="hero-section">
+          <h1>How can I Recover my Phone if it is lost?</h1>
```
Comment (bhavesh92pandey) [inline pages/blog/how-can-i-recover-my-phone.js:13]: This is the Title (the font should be bigger than H1)

Comment (amitg01) [reply pages/blog/how-can-i-recover-my-phone.js:13]: Adjusted the font-size of the H1 to make the title bigger.

**Review by bhavesh92pandey — COMMENTED** [review]
Hi @amitg01 

Made some changes to the headings and subheadings. Please refer to the Blog document (link: https://docs.google.com/document/d/1fPiWW5K0_R2MjJ84CW8BRGEOuRPYvAQJWWECk3GGytI/edit#heading=h.8ftuju6tapbj)

For reference check: https://medium.com/ and blog.kleros.com

**Comment by bhavesh92pandey** [comment]
Hi @amitg01 
Do we have any update on this pull request?

**Comment by amitg01** [comment]
Hi Bhavesh

I was a bit occupied with some other work. I have made the necessary
changes. Just some small things are yet to be done. The PR will be ready by
E.O.D. today.

**Comment by amitg01** [comment]
@bhavesh92pandey The changes are done. Please review.

**Comment by bhavesh92pandey** [comment]
Hi @amitg01 

I have mailed you the photographs for Blog 2 cover page and blog 2 body. Please check. Also check the blog pdf for numbering and placement

**Comment by amitg01** [comment]
@bhavesh92pandey I have added the images to the blog index page. Please have a look.

File: pages/blog/how-can-i-recover-my-phone.js [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+              <li>
+                Log in to lost Windows device service to remotely lock your
+                phone or erase its contents.
+              </li>
+            </ul>
+          </div>
+        </section>
+        <section className="section-3">
+          <div className="measures-div">
+            <h2>Immediate measures to take</h2>
```
[the remark this answers is not in the document]
Comment (bhavesh92pandey) [reply pages/blog/how-can-i-recover-my-phone.js:159]: Screen Shot 2020-11-20 at 9 19 25 AM

File: pages/blog/how-can-i-recover-my-phone.js [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+            <ul>
+              <li>
+                <h3>Call the Police:</h3>
```
[the remark this answers is not in the document]
Comment (bhavesh92pandey) [reply pages/blog/how-can-i-recover-my-phone.js:162]: Screen Shot 2020-11-20 at 9 19 25 AM

Check the numbers, see if they can go with the heading.

[the remark this answers is not in the document]
Comment (amitg01) [reply pages/blog/how-can-i-recover-my-phone.js:162]: @bhavesh92pandey Done. Please have a look.

**Pull request merged**